### PR TITLE
ZMQ server

### PR DESCRIPTION
**Pull request by stefanoborini**
Adds zeromq server to handle communication with the execution

### CODE REVIEW

File: force_wfmanager/plugins/ui_notification/ui_notification_model.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+from traits.api import Unicode
 from force_bdss.api import (
-    BaseNotificationListenerModel, ZMQSocketURL)
+    BaseNotificationListenerModel)
 
 
 class UINotificationModel(BaseNotificationListenerModel):
     #: The socket URL where the UI will be found. Synchronization port.
-    sync_url = ZMQSocketURL()
+    sync_url = Unicode()
```
Comment (martinRenou) [inline force_wfmanager/plugins/ui_notification/ui_notification_model.py]: Why did you remove this `ZMQSocketURL ` custom trait ?

Comment (stefanoborini) [inline force_wfmanager/plugins/ui_notification/ui_notification_model.py]: restored
